jax/version.py

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.2.25"
+__version__ = "0.2.26"
```

Comment:
We want this updated after the release commit, yes?

Reply:
Oh, my mistake - this is correct, but the `0.2.25 (Unreleased)` above should be `0.2.26 (Unreleased)` – that threw me off.

Reply:
yes, updated, thanks for the catch